src/app/pages/AddPage/AddPage.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from 'react';
+import Header from '../../components/Header/Header';
+import GearSelector from '../../components/GearSelector/GearSelector';
+import GearInput from '../../components/GearInput/GearInput';
+import SaveButton from '../../components/SaveButton/SaveButton';
+import NavBar from '../../components/NavBar/NavBar';
+import style from './AddPage.module.css';
+
+export default function AddPage(): JSX.Element {
+  const [gearName, setGearName] = useState<string>('');
+
+  const [selectedGear, setSelectedGear] = useState<'Drum' | 'Synth' | 'Effect'>(
+    'Synth'
+  );
+
+  const [output, setOutput] = useState<string>('');
+  const [isOutputChecked, setIsOutputChecked] = useState(false);
+
+  const [input, setInput] = useState<string>('');
+  const [isInputChecked, setIsInputChecked] = useState(false);
+
+  const [midi_in, setMidi_in] = useState<string>('');
+  const [isMidi_inChecked, setIsMidi_inChecked] = useState(false);
+
+  const [midi_out, setMidi_out] = useState<string>('');
+  const [isMidi_outChecked, setIsMidi_outChecked] = useState(false);
+
+  const [usb, setUsb] = useState<string>('');
+  const [isUsbChecked, setIsUsbChecked] = useState(false);
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    const newGear = {
+      iconType: setSelectedGear,
+      name: gearName,
+      connections: [
+        { channel: output, connection: 'Output to' },
+        { channel: input, connection: 'Input from' },
+        { channel: midi_in, connection: 'MIDI In from' },
+        { channel: midi_out, connection: 'MIDI Out from' },
+        { channel: usb, connection: 'USB Port' },
```

Comment:
This should be filled conditionally based on the checked value of the field

Reply:
Resolved in [e41db93](https://github.com/Seborider/plugs./pull/39/commits/e41db93f3c0e7abf26e9e9ecdd7c867c346724ea)